dev/io.openliberty.microprofile.telemetry.1.0.internal.jaeger_fat/test-bundles/com.ibm.ws.opentracing.zipkin.tracer/src/com/ibm/ws/opentracing/zipkin/tracer/OpentracingZipkinTracerFactory.java

```diff
@@ -0,0 +1,86 @@
+/*****************************************************************************
+ * Copyright (c) 2017, 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * IBM Corporation - initial API and implementation
+ *****************************************************************************/
+package com.ibm.ws.opentracing.zipkin.tracer;
+
+import java.util.Map;
+
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+import org.osgi.service.component.annotations.Modified;
+
+import aQute.bnd.annotation.metatype.Configurable;
+import aQute.bnd.annotation.metatype.Meta;
+import io.openliberty.opentracing.spi.tracer.OpentracingTracerFactory;
+import io.opentracing.Tracer;
+import zipkin2.codec.Encoding;
+
+/**
+ * Factory for delivering Opentracing Tracers backed by Zipkin implementation.
+ */
+
+@SuppressWarnings("restriction")
+//Annotation is used for the creation of the OSGI bundle manfiest information.
+@Component(immediate = true, configurationPolicy = ConfigurationPolicy.OPTIONAL, configurationPid = "opentracingZipkin")
+public class OpentracingZipkinTracerFactory implements OpentracingTracerFactory {
```

Comment:
I thought this code was included in the the user feature jar. Why do you also have the source here?